packages/components/Callout/src/Callout.types.ts

```diff
@@ -97,6 +97,13 @@ export interface ICalloutProps extends ICalloutTokens {
    */
   onShow?: () => void;
 
+  /**
+   * If true then the callout will attempt to focus the first focusable element that it contains.
+   * If it doesn't find an element, no focus will be set and the method will return false.
```

Comment:
"...no focus will be set and the method will return false."

There's no method here. :)